packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -65,23 +65,50 @@ export const Avatar = (props: AvatarProps) => {
   // TODO: Revise size handling. Do we need to be this dynamic or should we use the theme instead?
   return (
     <Flex
-      elementDescriptor={[boxElementDescriptor || descriptors.avatarBox]}
+      elementDescriptor={[boxElementDescriptor, descriptors.avatarBox]}
       sx={[
-        theme => ({
+        t => ({
           flexShrink: 0,
-          borderRadius: rounded ? theme.radii.$circle : theme.radii.$md,
+          borderRadius: rounded ? t.radii.$circle : t.radii.$md,
           overflow: 'hidden',
-          width: size(theme),
-          height: size(theme),
-          border: theme.borders.$normal,
-          borderColor: theme.colors.$avatarBorder,
-          backgroundColor: theme.colors.$avatarBackground,
+          width: 'var(--cl-avatar-size)',
+          height: 'var(--cl-avatar-size)',
+          backgroundColor: t.colors.$avatarBackground,
           backgroundClip: 'padding-box',
+          position: 'relative',
+          boxShadow: 'var(--cl-shimmer-hover-shadow)',
```

Comment:
How can I disable the shadow without disabling the shimmer effect?

Reply:
You cannot. A workaround is to disable the shadow completely for the avatar element. 
We could add 2 layout variables, but I think this is too much for now.